include/sys/epoll.h

```diff
@@ -91,11 +94,14 @@ enum
 #define EPOLL_CLOEXEC EPOLL_CLOEXEC
 };
 
-typedef union poll_data
+typedef union epoll_data
 {
   FAR void    *ptr;
   int          fd;
   uint32_t     u32;
+#ifdef CONFIG_HAVE_LONG_LONG
+  uint64_t     u64;
+#endif
 } epoll_data_t;
```

Comment:
can we change to
```
union epoll_data
{
  FAR void    *ptr;
  int          fd;
  uint32_t     u32;
#ifdef CONFIG_HAVE_LONG_LONG
  uint64_t     u64;
#endif
};

typedef union epoll_data epoll_data_t;
```
?
I think that is what is used in most of the files

Reply:
Done.